tests/config/test_config.py

```diff
@@ -15,6 +17,21 @@ class LoadConfigTest(unittest.TestCase):
     def test_load_nothing(self):
         self.assertEqual({}, config._load([], [], []))
 
+    def test_load_missing_file(self):
+        file0 = path_to_data_dir('file0.conf')
+        result = config._load([file0], [], [])
+        self.assertEqual({}, result)
+
+    def test_load_nonreadable_file(self):
+        @patch('os.access')
```

Comment:
You can put this decorator on the `test_load_nonreadable_file` method. Also, `@mock.patch(...)` works.